include/veriblock/blockchain/blocktree.hpp

```diff
@@ -299,13 +305,13 @@ struct BlockTree {
       return state.addStackFunction("acceptBlock");
     }
 
-    auto* index = insertBlockHeader(block);
+    auto index = insertBlockHeader(block);
```

Comment:
You can write directly to blockIndex, eg &blockIndex = insertBlockHeader(block);